### PR TITLE
fix: Issue with boxes displaying as icons in EFB map

### PR DESCRIPTION
**Pull request by antonyisoardi**
Issue: https://github.com/flybywiresim/a32nx/issues/4761

A new ControlPanel has been created which includes the existing functions - Polyline measurement, clear polylines, change units of measurement, zoom in and zoom out.

The issue with the icons not displaying correctly was that they weren't icons at all, rather just text characters. Some of these characters were "special" and the glyphs weren't actually present in whichever font was being used in MSFS. To ensure this doesn't happen again, I've used SVG icons from @tabler/icons in the new ControlPanel.

While I couldn't do anything with changing the controls in the polylineMeasure control (which extends off leaflet controls), I've kept it in place and am calling functions against it once instantiated.

localhost_6006__path=_story_map--force-carto-light

Will refactor the A320x EFB once this is merged.

### CODE REVIEW

File: src/Map/Map.scss [hunk 1]
```diff
@@ -10,6 +10,21 @@
   // Prevent leaflet from setting its own font
   font-family: "Nunito Sans", sans-serif !important;
 
+  #control-panel {
```
Comment (Benjozork) [inline src/Map/Map.scss:13]: Use tailwind as much as possible

**Review by Benjozork — CHANGES_REQUESTED** [review]

**Comment by Benjozork** [comment]
Make sure you don't have purging enabled

**Review by Benjozork — APPROVED** [review]

**Comment by antonyisoardi** [comment]
> Make sure you don't have purging enabled

Hey @Benjozork - can you please elaborate on this one?

**Comment by Benjozork** [comment]
If some tailwind classes don't work you might have purging enabled.

**Comment by antonyisoardi** [comment]
@Benjozork what else needs to happen for this to get merged?

**Comment by Benjozork** [comment]
I'll test it locally today and get it merged.

**Comment by Benjozork** [comment]
@antonyisoardi Found the issue with classes not working.

Purging is force-enabled:

image

Since purging is not run on every webpack build, previously unused classes are not kept.

I'll make a PR to change this after merging this one.

This did not affect production builds.

**Pull request merged**